OrbitCore/Capture.cpp

```diff
@@ -60,6 +60,7 @@ void Capture::FinalizeCapture() {
 void Capture::ClearCaptureData() {
   GSelectedTextBox = nullptr;
   GSelectedThreadId = 0;
+  capture_data_ = CaptureData();
```

Comment:
This will break functionality due to recent changes.
ClearCaptureData gets invoked basically from two places:
1. `OnLoadCapture`, invokes this, and then calls `CaptureSerializer` to load the capture (which will set capture data again). So this is no harmful change here, but still not needed, as it will be reset a couple of instructions later.

2. `OnCaptureStarted`: This gets invoked by the `CaptureClient` **after** the `App::StartCapture` resets the `capture_data_`. Thus all information (like the selected_functions), will be gone. Again, this reset would not be needed, as StartCapture, sets new capture data.

Reply:
> This will break functionality due to recent changes.

Yes it seems like it does. I'll look into it, thanks!